.eslintrc.js

```diff
@@ -143,6 +143,7 @@ module.exports = {
                             "!matrix-js-sdk/src/models/read-receipt",
                             "!matrix-js-sdk/src/models/relations-container",
                             "!matrix-js-sdk/src/models/related-relations",
+                            "!matrix-js-sdk/src/models/invites-ignorer",
```

Comment:
surely we shouldn't be adding new exclusions here, unless we're deliberately defining new entrypoints for the js-sdk?

Reply:
The invites-ignorer stuff is an experimental MSC implementation, its not re-exported presumably for the reason of us not wanting to treat it as stable yet

Reply:
Moved to a one-off lint override to not set a bad standard